### DISCUSSION TITLE
zh/guestbook

### DISCUSSION THREAD

**Discussion by giscus** [orangejuice/blog#4] [Announcements]
**English**: https://orangejuice.cc/guestbook
**中文**: https://orangejuice.cc/zh/guestbook

**Comment by orangejuice** [comment]
Hello world! 🗺️

**Comment by orangejuice** [comment]
2024.6.18

**Comment by tommm2** [comment]
nice blog

**Reply by orangejuice** [reply]
Thank you 😊

**Comment by lllOrz** [comment]
复古老梗：踩一下，记得回踩哦（虽然我没有博客🤣）

**Reply by orangejuice** [reply]
😁 一句话直接穿越回2000年代 （快去写一个）

**Reply by lllOrz** [reply]
写不了一点儿，下班除了吃就是玩儿

**Comment by lllOrz** [comment]
行，以后找电影看就来你这儿

**Reply by orangejuice** [reply]
😆 有好看的电影记得推荐给我
